csrc/device_lower/utils.h

```diff
@@ -16,7 +16,6 @@
 #include <kernel_ir.h>
 #include <parallel_type_bitmap.h>
 #include <val_graph.h>
-#include <val_graph_visitor.h>
```

Comment:
Cleaned up the usage of this header file as I noticed just changing this file required recompilation of more than 160 files..